general/development/policies/codingstyle/frankenstyle.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Frankenstyle
```

Comment:
I might suggest renaming the page title to 'Frankenstyle component names'

To keep the sidebar label short, we can use the `sidebar_label:` yaml attribute: https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-docs#sidebar_label

So:
```suggestion
title: Frankenstyle Component Names
sidebar_label: Frankenstyle
```